orbit-2-test/src/main/java/com/babylon/orbit2/TestFlowObserver.kt

```diff
@@ -1,30 +1,33 @@
 package com.babylon.orbit2
 
-import kotlinx.coroutines.CoroutineScope
+import kotlinx.atomicfu.atomic
+import kotlinx.atomicfu.getAndUpdate
 import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.cancel
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.Job
 import kotlinx.coroutines.delay
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.collect
 import kotlinx.coroutines.launch
 import kotlinx.coroutines.runBlocking
 import kotlinx.coroutines.withContext
+import java.io.Closeable
```

Comment:
using java.io will need to change for KMM, no?

Reply:
GOOD SPOT

Reply:
Use from https://github.com/Kotlin/kotlinx-io?

Reply:
Having discussed it probably overkill to include a whole library for just one interface. This is not a huge priority so deprioritising for now